index.html

```diff
@@ -5,25 +5,39 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;800&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./static/style.css">
-  <title>Disktop_Versioin_Portofolio</title>
+  <title>Mahram's_Portofolio</title>
 </head>
 <body> 
   <header> 
+    <div id="modal__container" class="modal__container">
+      <div class="modal__header">
+        <div></div>
+        <i id="btn__close" class="fa fa-times modal__icon" aria-hidden="true"></i>
+      </div>
+      <div class="modal__content">
+        <h2 id="modal__titles1"> Portofolio</h2>
+        <h2 id="modal__titles2"> About</h2>
+        <h2 id="modal__titles3"> Contact</h2>
+      </div>
```

Comment:
- [ ] As specified in the project requirements please make sure that when the user clicks any of the mobile menu options, the correct part of the page is displayed. So that means the above headings should be a link tag that should redirect to the corresponding section of the page. ✔
- Here's the project requirement as a reminder:
  ![image](https://user-images.githubusercontent.com/104968236/232763938-3fca98ed-af1c-439d-a112-5fc2c5159f17.png)